tensorflow/core/ops/state_ops.cc

```diff
@@ -260,6 +278,26 @@ REGISTER_OP("ScatterNdSub")
     .Attr("use_locking: bool = false")
     .SetShapeFn(shape_inference::ScatterNdUpdateShape);
 
+REGISTER_OP("ScatterNdMin")
```

Comment:
I'd not bother registering these, as non-resource variables won't be really supported after tf2.0